### PR TITLE
MM-15295: migrate commandstore.AnalyticsCommandCount to sync by default

### PR DESCRIPTION
**Pull request by therealpuneeth20**
#### Summary
This PR migrates commandstore.AnalyticsCommandCount to sync by default

#### Ticket Link

  Fixes https://github.com/mattermost/mattermost-server/issues/10722

### CODE REVIEW

File: store/sqlstore/command_store.go [hunk 1]
```diff
@@ -150,24 +150,22 @@ func (s SqlCommandStore) Update(cmd *model.Command) store.StoreChannel {
...
 }
 
-func (s SqlCommandStore) AnalyticsCommandCount(teamId string) store.StoreChannel {
-	return store.Do(func(result *store.StoreResult) {
-		query :=
-			`SELECT
+func (s SqlCommandStore) AnalyticsCommandCount(teamId string) (int64, *model.AppError) {
+	query :=
+		`SELECT
 			    COUNT(*)
```
Comment (jespino) [inline store/sqlstore/command_store.go]: Keep the original indentation relation here inside the string, please.

Comment (jespino) [reply store/sqlstore/command_store.go]: wow, that was quick :)

File: app/diagnostics.go [hunk 2]
```diff
@@ -193,8 +193,8 @@ func (a *App) trackActivity() {
 		postsCount = pcr.Data.(int64)
 	}
 
-	if scc := <-a.Srv.Store.Command().AnalyticsCommandCount(""); scc.Err == nil {
-		slashCommandsCount = scc.Data.(int64)
+	if c, err := a.Srv.Store.Command().AnalyticsCommandCount(""); err == nil {
+		slashCommandsCount = c
```
Comment (grundleborg) [inline app/diagnostics.go]: Could you just directly assign to `slashCommandsCount` here?

Comment (therealpuneeth20) [reply app/diagnostics.go]: yeah, will update it.

**Pull request merged**